lib/cli/interactive-setup/index.js

```diff
@@ -12,15 +12,20 @@ const steps = {
 };
 
 module.exports = async (context) => {
-  context = { ...context, inquirer, history: new Map() };
+  context = { ...context, inquirer, history: new Map(), stepDurations: new Map() };
   for (const [stepName, step] of Object.entries(steps)) {
     delete context.stepHistory;
     const stepData = await step.isApplicable(context);
+    const stepStartTime = process.hrtime();
     if (stepData) {
       process.stdout.write('\n');
-      context.stepHistory = [];
+      context.stepHistory = {};
```

Comment:
I think it'll be good to maintain the order, so I guess I'll keep it as an array

While now we agreed that we will just present properties in mixpanel, I wouldn't be surprised if at some point we'll get the request, to add an array property, that shows the order of processed steps (as assuming it from object properties will be really painful)

Maybe from a start, what we should send with telemetry should be something as:

```javascript
commandUsage: [
  [propertyName", "propertyValue"]
]
```

And then for mixpanel we may translate it into convient form, so create some `step_propertyName` but also add `interactiveCLI_order` array, in which we will present the order of each steps (?)

Reply:
I started with array-based approach, but processing was cumbersome so I've decided to switch to more convenient, object-based way. I figured that we lose ordering anyway in Mixpanel, but I see the reasoning and I'm fine with reverting the changes

Reply:
Do you feel like we should also explicitly record the order to `steps`, not only questions? Because that will require further changes to the shape of `history`

Reply:
> Because that will require further changes to the shape of history

`Map` is _ordered_ per spec (even if conceptually it isn't), so it's already ordered

Reply:
I've changed the implementation to send the data in shape that preserves ordering of both steps as well as questions within steps. Please let me know what do you think